addons/addon-base-raas/packages/base-raas-rest-api/lib/controllers/budgets-controller.js

```diff
@@ -38,7 +38,7 @@ async function configure(context) {
   // ===============================================================
   //  PUT / (mounted to /api/budgets)
```

Comment:
Please change this line to the following:
`//  POST / (mounted to /api/budgets)`